schemas/Screen.json

```diff
@@ -46,11 +42,28 @@
         {"const": "MobileSwitchDirectory", "description": "The screen shown to select which room directory you'd like to use."},
         {"const": "MobileInviteFriends", "description": "The screen shown to share a link to download the app."},
         {"const": "MobileBreadcrumbs", "description": "The screen that displays the user's breadcrumbs."},
+        {"const": "MobileSettings", "description": "The global settings screen shown in the app."},
+        {"const": "MobileSettingsSecurity", "description": "The global security settings screen."},
+        {"const": "MobileSettingsDefaultNotifications", "description": "The settings screen to change the default notification options."},
+        {"const": "MobileSettingsMentionsAndKeywords", "description": "The settings screen to manage notification mentions and keywords."},
         
-        {"const": "WebLoading", "description": "?"},
-        {"const": "WebCompleteSecurity", "description": "?"},
-        {"const": "WebE2ESetup", "description": "?"},
-        {"const": "WebSoftLogout", "description": "?"}
+        {"const": "WebLoading", "description": "Element Web loading spinner."},
+        {"const": "WebCompleteSecurity", "description": "Element Web showing flow to trust this new device with cross-signing."},
+        {"const": "WebE2ESetup", "description": "Element Web showing flow to setup SSSS / cross-signing on this account."},
+        {"const": "WebSoftLogout", "description": "Element Web device has been soft logged out by the server."},
+        {"const": "WebUserSettingsGeneral", "description": "Element Web User Settings General Tab."},
+        {"const": "WebUserSettingsAppearance", "description": "Element Web User Settings Appearance Tab."},
+        {"const": "WebUserSettingsNotifications", "description": "Element Web User Settings Notifications Tab."},
+        {"const": "WebUserSettingsPreferences", "description": "Element Web User Settings Preferences Tab."},
+        {"const": "WebUserSettingsKeyboard", "description": "Element Web User Settings Keyboard Tab."},
+        {"const": "WebUserSettingsSidebar", "description": "Element Web User Settings Sidebar Tab."},
+        {"const": "WebUserSettingsVoiceVideo", "description": "Element Web User Settings Voice & Video Tab."},
+        {"const": "WebUserSettingsSecurityPrivacy", "description": "Element Web User Settings Security & Privacy Tab."},
+        {"const": "WebUserSettingsLabs", "description": "Element Web User Settings Labs Tab."},
+        {"const": "WebUserSettingsIgnoredUsers", "description": "Element Web User Settings Ignored Users Tab."},
+        {"const": "WebUserSettingsHelpAbout", "description": "Element Web User Settings Help & About Tab."},
+        {"const": "WebUserSettingMjolnir", "description": "Element Web User Settings Mjolnir (labs) Tab."},
+        {"const": "WebUserSettingFlair", "description": "Legacy: Element Web User Settings Flair Tab."}
```

Comment:
See my remark on the other PR. I do not understand why you use `Web` prefix for all those screens. Most of them exist on Element Android

Reply:
Also the philosophy is to share the maximum of events between the 3 Element clients, this is why this project exist. So using prefix is against this :)

Reply:
But they have different contents between the platforms, so if someone using Posthog wanted to see How Many users saw setting X it may be in different places between Web & Mobile. So the query would get needlessly more complicated.

Reply:
I agree this is a problem. I think Gaelle is working to improve the parity of the settings between the 3 clients.